kade-net/lagoon
# Ft/update address script

- Ignoring files used for updating tables
- Updates in schema

To test the update table script
1. Run migrations
2. Set PG_CONNECTION_STRING to local postgres instance
3. Run fill-test-values.ts script to update local postgres instance with test data, the script contains info on what addresses are meant to changes
4. Run the update-table.ts script
5. View local db to see changes

To run script on testnet or production
1. Run migrations 
2. Set PG_CONNECTION_STRING to that of tesnet or prod
3. Run update-tables.ts script
